website/source/docs/platform/k8s/helm.html.md

```diff
@@ -377,6 +377,176 @@ to run the sync program.
             "envoy_dogstatsd_url": "udp://127.0.0.1:9125"
           }
         ```
+* <a name="v-meshgateway" href="#v-meshgateway">`meshGateway` <sup>(beta)</sup></a> - Values that configure Connect [Mesh Gateways](/docs/connect/mesh_gateway.html).
+
+  * <a name="v-meshgateway-enabled" href="#v-meshgateway-enabled">`enabled`</a> (`boolean: false`) - If true, gateways will be created and
+    Consul servers will be configured to support them. If setting this in an already running cluster, you'll need to
+    reload the servers.
+
+  * <a name="v-meshgateway-globalmode" href="#v-meshgateway-globalmode">`globalMode`</a> (`string: "local"`) -
+  [Globally configure](/docs/connect/mesh_gateway.html#enabling-gateways-globally) which mode the gateway should run in.
+  Can be set to either "remote", "local", "none" or "".
+  See [Modes of Operation](/docs/connect/mesh_gateway.html#modes-of-operation) for a description of each mode.
+  If set, `connectInject.centralConfig.enabled` should be set to true so that
+  the global config will actually be used.
+  If set to the empty string, no global default will be set and the gateway mode
+  will need to be set individually for each service.
+
+  * <a name="v-meshgateway-replicas" href="#v-meshgateway-replicas">`replicas`</a> (`int: 2`) -
+  How many gateway pods to run.
```

Comment:
```suggestion
  How many gateway Pods to run.
```

Reply:
Sorry I thought that "Pod" would be lowercase but it looks like they capitalize it in the K8s docs so I guess we should do the same.